src/engraving/libmscore/volta.cpp

```diff
@@ -22,18 +22,19 @@
 
 #include "volta.h"
 
+#include <algorithm>
+
+#include "style/style.h"
+
```

Comment:
do we need all these extra blank lines?

Reply:
I usually try to group includes
but this is not necessary, optional